bin/node_init

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# LEAP Platform node initialization.
+# This script is run on the target server when `leap node init` is run.
+#
+
+DEBIAN_VERSION="^(jessie|8\.)"
+LEAP_DIR="/srv/leap"
+HIERA_DIR="/etc/leap"
+INIT_FILE="/srv/leap/initialized"
+REQUIRED_PACKAGES="puppet rsync lsb-release locales"
+
+APT_GET="apt-get -q -y -o DPkg::Options::=--force-confold"
+APT_GET_UPDATE="apt-get update -o Acquire::Languages=none"
+BAD_APT_RESPONSE="(BADSIG|NO_PUBKEY|KEYEXPIRED|REVKEYSIG|NODATA|Could not resolve|failed to fetch)"
+export DEBIAN_FRONTEND=noninteractive
+
+test -f $INIT_FILE && rm $INIT_FILE
+if ! egrep -q "$DEBIAN_VERSION" /etc/debian_version; then
+  echo "ERROR: This operating system is not supported. The file /etc/debian_version must match /$DEBIAN_VERSION/ but is: `cat /etc/debian_version`"
+  exit 1
+fi
+mkdir -p $LEAP_DIR
+echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
+
+#
+# UPDATE PACKAGES
+# (exit code is not reliable, sadly)
+#
+echo "updating package list"
+
+error_count=0
+while read line; do
+  error=$(echo $line | egrep "$BAD_APT_RESPONSE")
+  if [[ $error ]]; then
+    errors[error_count]=$error
+    ((error_count++))
+    break # should we halt on first error?
+  fi
+  echo $line
+done < <($APT_GET_UPDATE 2>&1)
+
+if [[ $error_count > 0 ]]; then
+  echo "ERROR: fatal error in 'apt-get update', bailing out."
+  for e in "${errors[@]}"; do
+    echo "       $e"
+  done
+  exit 1
+fi
+
+#
+# UPDATE TIME
+#
+echo "updating server time"
+test -f /etc/init.d/ntp && /etc/init.d/ntp stop
```

Comment:
For jessie, this should be:
/bin/systemctl -q is-active ntp.service && /bin/systemctl stop ntp.service